services/storage/src/simcore_service_storage/dsm.py

```diff
@@ -441,7 +442,7 @@ async def try_update_database_from_storage(
         file_uuid: str,
         bucket_name: str,
         object_name: str,
-        silence_exception: bool = False,
+        reraise_exceptions: bool = False,
```

Comment:
It was probably me in the first place, but could you make ```reraise_exceptions``` an argument **without** default value? I find this very unclear now. thanks